beacon_node/eth1/src/http.rs

```diff
@@ -91,14 +92,22 @@ pub async fn get_network_id(endpoint: &str, timeout: Duration) -> Result<Eth1Id,
 
 /// Get the eth1 chain id of the given endpoint.
 pub async fn get_chain_id(endpoint: &str, timeout: Duration) -> Result<Eth1Id, String> {
-    let response_body = send_rpc_request(endpoint, "eth_chainId", json!([]), timeout).await?;
-    hex_to_u64_be(
-        response_result(&response_body)?
-            .ok_or("No result was returned for chain id")?
-            .as_str()
-            .ok_or("Data was not string")?,
-    )
-    .map(Into::into)
+    let response_body: String =
+        send_rpc_request(endpoint, "eth_chainId", json!([]), timeout).await?;
+
+    /* extract response text here */
```

Comment:
It's nitpick, but comments in the LH codebase generally look like this:

```suggestion
    // Extract response text here.
```

Reply:
Consult 31be1ab.